h/storage.py

```diff
@@ -132,14 +132,8 @@ def create_annotation(request, data, group_service):
                                       _('You may not create annotations '
                                         'in the specified group!'))
 
-    if request.feature('filter_groups_by_scope') and group.scopes:
-        # The scope (origin) of the target URI must match at least one
-        # of a group's defined scopes, if the group has any
-        group_scopes = [scope.origin for scope in group.scopes]
-        if not group_scope_match(data['target_uri'], group_scopes):
-            raise schemas.ValidationError('group scope: ' +
-                                          _('Annotations for this target URI '
-                                            'are not allowed in this group'))
+    if request.feature('filter_groups_by_scope'):
```

Comment:
This feature flag has now been turned on for everyone since we released this, so I think we are at a point where we can remove it and simplify ongoing work?